compiler/model/metamodel.ts

```diff
@@ -240,6 +247,19 @@ export class Request extends BaseType {
   attachedBehaviors?: string[]
 }
 
+/**
+ * A response type
+ */
+export class Response extends BaseType {
```

Comment:
the properties:
* `statusCode: number | number[]`
* `headers: string[]`

are missing based on https://gist.github.com/delvedor/e388ca6b079fc2cb9bbb607fef3ced53#what-about-the-schema

Reply:
That was an example, we aren't tracking those yet as we should discuss more what to store there.
For example,  `statusCode` should be a simple number or a more complex structure that shows which type will be used based on the status?
As soon as we define what we want to have there, we can add new properties.

Reply:
👍